### PR TITLE
Feature bp 742 replace movai classes with the gd node package

### PR DESCRIPTION
**Pull request by erezz-mov-ai**
This branch contains all necessary changed to split movai_classes
jira:
https://movai.atlassian.net/browse/BP-742?atlOrigin=eyJpIjoiNjc1NDYxZTg0MTU5NDY2NjhmNTY4YmVlYjJmMzEwZDAiLCJwIjoiaiJ9
- [x] Make sure you are opening from a **topic/feature/bugfix branch**
- [x] Ensure that the PR title represents the desired changes
- [x] Ensure that the PR description detail the desired changes
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

[^note]:
    Put an `x` into the [ ] to show you have filled the information.
    The template comes from https://github.com/MOV-AI/.github/blob/master/.github/pull_request_template.md
    You can override it by creating .github/pull_request_template.md  in your own repository

### CODE REVIEW

File: gd_node/message.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 from movai_core_shared.envvars import ROS2_PATH
 
-from .message_model import Message
+from dal.scopes.message import Message
```
Comment (Mograbi) [inline gd_node/message.py:17]: from dal.scopes import Message

File: gd_node/protocols/movai.py [hunk 2]
```diff
@@ -16,9 +16,8 @@
 
 from dal.movaidb import RedisClient
 from dal.movaidb.database import MovaiDB
-from dal.scopes import Robot
-
-from gd_node.statemachine import SMVars, StateMachine
+from dal.scopes.robot import Robot
```
Comment (Mograbi) [inline gd_node/protocols/movai.py:19]: same

**Review by Mograbi — CHANGES_REQUESTED** [review]

File: gd_node/callback.py [hunk 3]
```diff
@@ -281,7 +280,7 @@ def __init__(self, sm_name: str):
                         "Task": Task,
                         "TaskTemplate": TaskTemplate,
                         "TaskEntry": TaskEntry,
-                        "Annotations": Annotations,
+                        "Annotation": Annotation,
                         "GraphicAsset": GraphicAsset,
                         "GraphicScene": GraphicScene,
                         "Layout": Layout,
```
Comment (erezz-mov-ai) [inline gd_node/callback.py:286]: add another self.global.update if enterprise

File: gd_node/callback.py [hunk 4]
```diff
@@ -254,8 +255,6 @@ def __init__(self, sm_name: str):
             self.globals.update(
                 {
                     "scopes": scopes,
-                    # hacky but quick way to export all models
-                    **{key: getattr(dal.models, key) for key in dal.models.__all__},
```
Comment (erezz-mov-ai) [inline gd_node/callback.py:258]: add all necessary models, mainly those used by robotics

**Review by Mograbi — APPROVED** [review]

**Review by Mograbi — APPROVED** [review]

**Pull request merged**